lib/grape/endpoint.rb

```diff
@@ -188,6 +188,7 @@ def compile_path(prepared_path, anchor = true, requirements = {})
     end
 
     def call(env)
+      build_app
```

Comment:
@dm1try @etehtsea personally i didn't want lazy initialization but when i put it in `initialize` there're failing test :(